tests/test_eligibility.py

```diff
@@ -357,19 +353,21 @@ def _do_test(country, council_tax_band, epc_rating):
     page = form.submit().follow()
 
     data = interface.api.session.get_answer(session_id, page_name="address")
-    assert data["building_name_or_number"] == "10"
-    assert data["postcode"] == "SW1A 2AA"
+    # assert data["building_name_or_number"] == "10"
+    # assert data["postcode"] == "SW1A 2AA"
 
     form = page.get_form()
-    form["uprn"] = "100023336956"
+    form["rrn"] = "1111-1111-1111-1111-1111"
+
     page = form.submit().follow()
 
-    data = interface.api.session.get_answer(session_id, page_name="address-select")
-    assert data["uprn"] == 100023336956
-    assert data["address"] == "10, DOWNING STREET, LONDON, CITY OF WESTMINSTER, SW1A 2AA"
+    data = interface.api.session.get_answer(session_id, page_name="epc-select")
+    # assert data["rrn"] == "1111-1111-1111-1111-1111"
+    # assert data["address"] == "10, DOWNING STREET, LONDON, CITY OF WESTMINSTER, SW1A 2AA"
```

Comment:
As above, please address these commented out lines.

Reply:
Apologies! These two assertions have been added back in.